src/SingleProject/Resizetizer/test/UnitTests/Resizetizer.UnitTests.csproj

```diff
@@ -8,13 +8,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.8.0" />
     <PackageReference Include="Microsoft.Build.Framework" Version="16.7.0" PrivateAssets="all" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="16.7.0" PrivateAssets="all" />
     <PackageReference Include="SkiaSharp" />
     <PackageReference Include="SkiaSharp.Extended" />
-    <PackageReference Include="xunit" Version="2.5.1" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.5.1" />
+    <PackageReference Include="xunit" Version="$(XunitPackageVersion)"/>
     <PackageReference Include="coverlet.collector" Version="6.0.0" />
     <PackageReference Include="Svg.Skia" />
```

Comment:
Is the `Microsoft.NET.Test.Sdk` package not needed? I see this is always added in the xunit templates.

ANd `xunit.runner.visualstudio`? Is that not needed for the IDE anymore?

Reply:
Nop, those are added by the arcade sdk